site/src/pages/RoadmapPage/Course.tsx

```diff
@@ -29,6 +29,7 @@ interface CourseNameAndInfoProps {
   requiredCourses?: string[];
   /** Whether to always collapse whitespace in the course name */
   alwaysCollapse?: boolean;
+  // change prev to isTile
```

Comment:
Hanging comment; decide whether you want to make the change, and then remove this comment.